paperqa/settings.py

```diff
@@ -612,6 +627,24 @@ class Settings(BaseSettings):
             " router_kwargs key with router kwargs as values."
         ),
     )
+    use_batch_in_summary: bool = Field(
+        default=False,
+        description=(
+            "Whether to use batch API for LLMs in summarization, "
+            "which means multiple messages are sent in one API request "
+            "to the LLM provider's batch API."
+            "This option is only available for Claude(https://docs.anthropic.com/en/api/creating-message-batches)"
+            "and OpenAI (https://platform.openai.com/docs/guides/batch) chat models."
+        ),
+    )
+    batch_summary_timelimit: int = Field(
+        default=24 * 60 * 60,
```

Comment:
Can you add a comment saying this matches OpenAI's default?

Reply:
Added it in the description